contrib/inventory/gce.py

```diff
@@ -247,6 +247,20 @@ def get_config(self):
             if states:
                 self.instance_states = states.split(',')
 
+        # Set the instance_tags filter
+        self.instance_tags = []
+        if config.has_option('gce', 'instance_tags'):
```

Comment:
Rather than using conditionals for each option, which can be a bit expensive, set a default value when the `config` object is created. Then you can just call `config.get()` and it will use the default if the key isn't in the file, or it will pull the value from the key in the file.